include/service/entities/article_service.js

```diff
@@ -191,6 +191,28 @@ ArticleService.prototype.find = function(where, options, cb) {
 };
 
 /**
+ * Updates articles
+ *
+ * @param {String} articleId	id of article
+ * @param {Object} options	
+ * @param {Object} fields	fields to update	
+ * @param {Function} cb      Callback function
+ */
+ArticleService.prototype.update = function(articleId, options, fields, cb) {
+	var where = {
+		_id: articleId
+	}
```

Comment:
This where clause is common across the code base.  For convenience, you can shortcut the syntax with: ```pb.DAO.getIdWhere(articleId)```